spylib/request.py

```diff
@@ -140,28 +160,10 @@ def make_service_request(
                 timeout=timeout,
                 retry_count=retry_count - 1,
             )
-        elif retry_count > 0 and resp.status_code == 401:
-            try:
-                decode(self.access_token, self.secret, algorithms=[self.algorithm])
-            except ExpiredSignatureError:
-                if not self.refresh_token:
-                    try:
-                        self.login(self.uuid, self.api_key)
-                    except LoginException:
-                        raise LoginException
-                else:
-                    self.access_token = self.refresh_access_token(self.refresh_token)
-                return self.make_service_request(
-                    base_url,
-                    path=path,
-                    method=method,
-                    payload=payload,
-                    timeout=timeout,
-                    retry_count=retry_count - 1,
-                )
+
         return resp
 
-    def delete(self, base_url, path, timeout, retry_count, **kwargs):
+    def delete(self, base_url, path, timeout=2, retry_count=0, **kwargs):
```

Comment:
🎈

Reply:
I had a dilemma/thought at some point... I wasn't sure whether or not we wanted to have default retries. 
How come we went for no default retry here? (I'm ok with this btw - just not sure why to opt for one way or the other ).